packages/core/src/domain/report/reportObservable.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { isChromium } from '../../tools/browserDetection'
+import type { Subscription } from '../../tools/observable'
+import { stubReportingObserver, stubCspEventListener } from '../../../test/specHelper'
+import { initReportObservable, CustomReportType } from './reportObservable'
+
+describe(`report observable`, () => {
+  let reportingObserverStub: { reset(): void; raiseReport(type: string): void }
+  let cspEventListenerStub: { dispatchEvent(): void }
+  let consoleSubscription: Subscription
+  let notifyReport: jasmine.Spy
+
+  beforeEach(() => {
+    if (!isChromium()) {
+      pending('no ReportingObserver support')
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Since we are stubing browser APIs for those tests, do we need to worry about whether the API is supported or not? I think it could be run in all browsers, no?

Reply:
Yes, why not :)